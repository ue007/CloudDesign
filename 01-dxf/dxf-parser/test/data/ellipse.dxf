0
SECTION
2
ENTITIES
0
ELLIPSE
5
37c
330
1F
100
AcDbEntity
8
0
6
ByLayer
100
AcDbEllipse
10
10.000000
20
10.000000
30
0.000000
11
10.000000
21
10.000000
31
0.000000
40
1.986299
41
0.000000
42
6.283185
0
ELLIPSE
5
37d
330
1F
100
AcDbEntity
8
0
6
ByLayer
100
AcDbEllipse
10
0.000000
20
-5.000000
30
0.000000
11
0.000000
21
5.000000
31
0.000000
40
4.000000
41
0.000000
42
6.283185
0
ENDSEC
0
EOF