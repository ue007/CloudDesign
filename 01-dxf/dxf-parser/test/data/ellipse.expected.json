{
  "entities": [
    {
      "type": "ELLIPSE",
      "handle": "37c",
      "ownerHandle": "1F",
      "layer": "0",
      "lineType": "ByLayer",
      "center": {
        "x": 10,
        "y": 10,
        "z": 0
      },
      "majorAxisEndPoint": {
        "x": 10,
        "y": 10,
        "z": 0
      },
      "axisRatio": 1.986299,
      "startAngle": 0,
      "endAngle": 6.283185
    },
    {
      "type": "ELLIPSE",
      "handle": "37d",
      "ownerHandle": "1F",
      "layer": "0",
      "lineType": "ByLayer",
      "center": {
        "x": 0,
        "y": -5,
        "z": 0
      },
      "majorAxisEndPoint": {
        "x": 0,
        "y": 5,
        "z": 0
      },
      "axisRatio": 4,
      "startAngle": 0,
      "endAngle": 6.283185
    }
  ]
}